tests/rogue_tests.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python
+PKG = 'herbpy'
+import roslib; roslib.load_manifest(PKG)
```

Comment:
This is a hold out from `rosbuild` days and is not necessary with Catkin. Please remove it.